corehq/apps/es/users.py

```diff
@@ -109,7 +109,8 @@ def user_ids(user_ids):
     return filters.term("_id", list(user_ids))
 
 
-def location(location_id):
+def primary_location(location_id):
+    # by primary location
```

Comment:
looks like you added that already in a later commit.  I think the project health dashboard should also be updated to that filter, though.